src/main/java/frc/robot/subsystems/flywheel/Flywheel.java

```diff
@@ -0,0 +1,167 @@
+package frc.robot.subsystems.flywheel;
+
+import static edu.wpi.first.units.Units.RPM;
+import static edu.wpi.first.units.Units.Volts;
+import static frc.robot.subsystems.flywheel.FlywheelConstants.*;
+
+import edu.wpi.first.units.measure.Voltage;
+import edu.wpi.first.wpilibj.Alert;
+import edu.wpi.first.wpilibj.Alert.AlertType;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import edu.wpi.first.wpilibj2.command.sysid.SysIdRoutine;
+import org.littletonrobotics.junction.Logger;
+
+public class Flywheel extends SubsystemBase {
+  private final FlywheelIO io;
+  private final FlywheelIOInputsAutoLogged inputs = new FlywheelIOInputsAutoLogged();
+
+  /* Disconnected alerts */
+  private final Alert flywheelDisconnectedAlert;
+
+  /* System variables and parameters */
+  private FlywheelState systemState = FlywheelState.IDLE;
+
+  private boolean requestLow = false;
```

Comment:
i dont' think you need these, you just need a state it's in and then something requests the next state via the enum?